airflow/providers/google/cloud/hooks/gcs.py

```diff
@@ -307,6 +307,79 @@ def is_updated_after(self, bucket_name, object_name, ts):
 
         return False
 
+    def is_updated_between(self, bucket_name, object_name, min_ts, max_ts):
+        """
+        Checks if an blob_name is updated in Google Cloud Storage.
+
+        :param bucket_name: The Google Cloud Storage bucket where the object is.
+        :type bucket_name: str
+        :param object_name: The name of the object to check in the Google cloud
+                storage bucket.
+        :type object_name: str
+        :param min_ts: The minimum timestamp to check against.
```

Comment:
I think we should be specific whether the min/max time is included or excluded. My proposal will be to 
include min_time but exclude max_time (min_time <= blob_update_time < max_time).
This will mimic the behaviour of [0:x] operator.

Reply:
Ok. I'll address it. Thanks